master.css

```diff
@@ -13,6 +13,18 @@ body {
 }
 table {
     margin-bottom: 2em; width: 100%;
+    border-spacing: 0;
+}
+thead {
+    position: -webkit-sticky;
+    position: -moz-sticky;
+    position: -ms-sticky;
+    position: -o-sticky;
```

Comment:
`position: sticky` doesn't work for me either (in FF). Might be because we're dealing with table-* elements here.